### PR TITLE
add comment out the definition of password encoder #370

### PR DESCRIPTION
**Pull request by btkobayashirun**
Please review #370.

### CODE REVIEW

File: src/main/resources/META-INF/spring/applicationContext.xml [hunk 1]
```diff
@@ -19,9 +19,11 @@
                 <entry key="bcrypt">
                     <bean class="org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder" />
                 </entry>
+                <!-- You need to add bcprov-jdk15on.jar to the dependency.
```
[the remark this answers is not in the document]
Comment (btkobayashirun) [reply src/main/resources/META-INF/spring/applicationContext.xml]: Fixed

501bd5a767e00d3f8b0a5fbcc31ee59eb537dbc2

File: src/main/resources/META-INF/spring/applicationContext.xml [hunk 2]
```diff
@@ -19,9 +19,11 @@
                 <entry key="bcrypt">
                     <bean class="org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder" />
                 </entry>
+                <!-- When using SCryptPasswordEncoder, you need to add bcprov-jdk15on.jar to the dependency.
                 <entry key="scrypt">
                     <bean class="org.springframework.security.crypto.scrypt.SCryptPasswordEncoder" />
                 </entry>
+                -->
```
[the remark this answers is not in the document]
Comment (btkobayashirun) [reply src/main/resources/META-INF/spring/applicationContext.xml]: The version is set indirectly specified by Spring-io.

Spring-io defines the version of bcpkix-jdk15on to 1.59, cprov-jdk15on is contained in bcpkix-jdk15on.

46549bf58a2ff0c7e77f4fe960cf58c3d24b9e7d

File: pom.xml [hunk 3]
```diff
@@ -244,6 +244,16 @@
         </dependency>
         <!-- == End Unit Test == -->
 
+        <!-- == Begin Password Encode == -->
+        <!--
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcprov-jdk15on</artifactId>
+            <version>${bouncycastle.version}</version>
+        </dependency>
```
[the remark this answers is not in the document]
Comment (btkobayashirun) [reply pom.xml]: add description

403ef5d49c89c0c8acbdb7a68c785846e2748a17

[the remark this answers is not in the document]
Comment (btkobayashirun) [reply pom.xml]: Move the definition after `End TERASOLUNA`

755f26d80fe73a0c637faf66de56e54a6c35bf83

File: pom.xml [hunk 4]
```diff
@@ -163,6 +163,17 @@
         </dependency>
         <!-- == End TERASOLUNA == -->
 
+        <!-- == Begin Password Encode == -->
```
Comment (yoshikawaa) [inline pom.xml]: - [ ] I think better to change section name `Password Encode` to `Security`. How are you think? @btkobayashirun @bttanakaysd

Comment (bttanakaysd) [reply pom.xml]: Agree.

**Pull request merged**